run_black.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# Convenience script for running black
+module use /g/data/v10/public/modules/modulefiles
+module load dea
+
+black -l 120  tests/ scene_select/
```

Comment:
Possibly could copy the `.pre-commit-config.yaml` from another repo. People always forget to run Black otherwise.

Reply:
Thanks for the suggestion.  I think i'll stay with hand cranking it for now.